common/djangoapps/student/forms.py

```diff
@@ -3,6 +3,9 @@
 """
 from importlib import import_module
 import re
+import socket
```

Comment:
I changed the old views because they were used with microsites in the dogwood release (and our production now is a fork of this release). I don't know if the old views are still used with microsites in the current master branch.